lib/frontend/templates/subtemplates/render_sentence.rs.html

```diff
@@ -0,0 +1,25 @@
+@use japanese::furigana::SentencePartRef;
+@type MyIterator<'a> = Vec<SentencePartRef<'a>>;
+@(iter: MyIterator)
+
+<div class="furigana-kanji-container"><!--
+-->@for furi_part in iter {<!--
+    -->@if furi_part.kana.len() != 0 {<!--
```

Comment:
We use `<!-- ... -->` to avoid having this entire block of code be on one line. We can't have any spaces between tags or Yomichan and copy/paste will not work.